noetic/distribution.yaml

```diff
@@ -3492,6 +3492,24 @@ repositories:
       url: https://github.com/ethz-adrl/ifopt.git
       version: master
     status: maintained
+  ign_ros_control:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/ign_ros_control.git
+      version: master
+    release:
+      packages:
+      - ign_ros_control
+      - ign_ros_control_demos
```

Comment:
I see there's a LICENSE file in `ign_ros_control`, mind adding another LICENSE file to the `ign_ros_control_demos` folder on the `master` branch? Alternatively you can have one LICENSE file at the top level of the repo. Please reply here when that's there and then this PR can be merged.

https://github.com/ros-controls/ign_ros_control/tree/master/ign_ros_control_demos

Reply:
Thanks for the note, I moved the LICENSE file to the root of the repo